admin_guide/manage_authorization_policy.adoc

```diff
@@ -0,0 +1,283 @@
+= Managing Authorization Policy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use the administrator CLI to manage
+link:../architecture/additional_concepts/authorization.html[authorization
+policy], such as adding and removing
+link:../architecture/additional_concepts/authorization.html#roles[roles] from
+users and groups.
+
+== Viewing Global Roles and Bindings
+As an administrator, you can view details about
+link:../architecture/additional_concepts/authorization.html#global-roles-and-bindings[global
+roles and bindings] using the `osc describe` command:
+
+.Describing the `default` Policy
+====
```

Comment:
@adellape I don't think we should turn commands into verbs. Please change the example title to what exactly your doing here. For example, "Viewing Global Roles and Bindings".